pkg/services/ngalert/models/alert_rule.go

```diff
@@ -469,6 +473,9 @@ func PatchPartialAlertRule(existingRule *AlertRule, ruleToPatch *AlertRule) {
 	if ruleToPatch.For == -1 {
 		ruleToPatch.For = existingRule.For
 	}
+	if !ruleToPatch.IsPaused {
```

Comment:
I don't think its possible to unpause an alert with `PatchPartialAlertRule`. If we will be using this method from the API then we won't be able to resume paused alerts?